examples/11_fs-router/src/pages/api/hello.ts

```diff
@@ -0,0 +1,7 @@
+export async function GET() {
+  return new Response('Hello from API!');
+}
+
+export async function POST(req: Request) {
+  return new Response(`Hello from API! ${new URL(req.url).pathname}`);
```

Comment:
My bad. That was my misunderstanding. I thought `req.url` is a URL object.

You can do this if you want. (Though, it shows "localhost", right?)

```suggestion
  return new Response(`Hello from API! ${req.url}`);
```

Reply:
Not very important in any case. Let's merge. You can change it afterwards if you want.

Reply:
I'll probably change this to show the body or something when adding other examples anyways